packages/modal/src/Modal.spec.tsx

```diff
@@ -58,9 +58,10 @@ describe('packages/modal', () => {
       const modal = getByRole('dialog');
       fireEvent.click(modal.parentElement!);
 
-      await expect(waitForElementToBeRemoved(modal)).rejects.toEqual(
+      await expect(waitForElementToBeRemoved(modal)).rejects.toBe(
         Error('Timed out in waitForElementToBeRemoved.'),
-      );
+      ); ///
```

Comment:
The error is slightly different now.

I'm seeing this: 
```
Expected value:  [Error: Timed out in waitForElementToBeRemoved.]
Received object: [Error: Timed out in waitForElementToBeRemoved.·
```

Reply:
cc: @lazytype

Reply:
my guess is node or jest dependency changed when the yarn.lock changed